packages/edit-site/src/components/header-edit-mode/document-actions/index.js

```diff
@@ -189,18 +186,12 @@ function BaseDocumentActions( { className, icon, children, onBack } ) {
 }
 
 function getEntityLabel( entityType ) {
-	let label = '';
-	switch ( entityType ) {
-		case 'wp_navigation':
-			label = 'navigation menu';
-			break;
-		case 'wp_template_part':
-			label = 'template part';
-			break;
-		default:
-			label = 'template';
-			break;
-	}
-
-	return label;
+	const labels = {
+		wp_block: __( 'Editing pattern:' ),
+		wp_navigation: __( 'Editing navigation menu:' ),
+		wp_template: __( 'Editing template:' ),
+		wp_template_part: __( 'Editing template part:' ),
+	};
+
+	return labels[ entityType ] ?? labels.wp_template;
```

Comment:
Because the default is template part when you edit a Page the title is not about editing a page but about editing a template part

<img width="1279" alt="Screen Shot 2023-07-27 at 16 20 05" src="https://github.com/WordPress/gutenberg/assets/444434/4e3cff4d-0aa4-426e-aeed-0f58623179a9">

Reply:
@getdave, I think you're looking at the wrong label in a11y tree. The pages use a different `Actions` component, which doesn't use this helper.

## Screenshot
![CleanShot 2023-07-27 at 22 35 43](https://github.com/WordPress/gutenberg/assets/240569/7494c66a-20ca-4fa4-99b0-b138cc1a6aba)